src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,99 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+
+// typography
+$pf-chart-global--FontSize--xs: $pf-global--FontSize--xs; // Axis Labels
+$pf-chart-global--FontSize--sm: $pf-global--FontSize--sm; // For Legend
+$pf-chart-global--FontSize--2xl: $pf-global--FontSize--2xl; // For Large Numbers
+
+// label
+$pf-chart-global-label--FontSize: $pf-chart-global--FontSize;
+$pf-chart-global-label--Padding: 10px;
+
+// Layout Props
+$pf-chart-global-layout--Padding: 50px;
+$pf-chart-global-layout--Height: 300px;
+$pf-chart-global-layout--Width: 450px;
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area-stroke--Width: 2px;
+
+// Axis Chart
+$pf-chart-axis-axis-stroke--Width: 1px;
+$pf-chart-axis-axis-label--Padding: 100px;
+$pf-chart-axis--ticks--Width: 1px;  // this should be a var for size so not sure if width is appropriate
+
+// Bar Chart
+$pf-chart-bar--Width: 10px;
```

Comment:
Do we want a var for stroke? I have this set to none, but we could easily change that via the theme. 

This is the color around the bar. Below I set this to yellow.

![Screen Shot 2019-05-29 at 11 03 57 AM](https://user-images.githubusercontent.com/17481322/58568041-820d5200-8201-11e9-917d-1b8e4abfbba4.png)

Reply:
Added a var